mir_eval/melody.py

```diff
@@ -294,6 +317,12 @@ def to_cent_voicing(ref_time, ref_freq, est_time, est_freq, base_frequency=10.,
         Time of each estimated frequency value
     est_freq : np.ndarray
         Array of estimated frequency values
+    est_voicing : np.ndarray
+        Estimate voicing confidence.
+        Default None, which means the voicing is inferred from ref_freq
```

Comment:
would be good to say here how this default inference happens

Reply:
I just added this actually! - see my push from 5 min ago

Reply:
There's even a bug in this docstring... should be "..is inferred from est_freq". I've updated and clarified!

Reply:
great! i like this explanation 👍